service/controller/app/v1/resource/chart/create.go

```diff
@@ -0,0 +1,28 @@
+package chart
+
+import (
+	"context"
+	"fmt"
+	"github.com/giantswarm/app-operator/service/controller/app/v1/key"
+	"github.com/giantswarm/microerror"
+	"k8s.io/apimachinery/pkg/api/errors"
```

Comment:
```suggestion
	apierrors "k8s.io/apimachinery/pkg/api/errors"
	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

These are the usual aliases we use. Also the imports need sorting.